include/flatbuffers/minireflect.h

```diff
@@ -332,6 +332,30 @@ inline std::string FlatBufferToString(const uint8_t *buffer,
   return tostring_visitor.s;
 }
 
+const char *LookUpAttribute(RawAttributeList list, const std::string &key) {
```

Comment:
make this argument a `const char *` to not force unnecessary string allocations on the user.